server/webhook.go

```diff
@@ -390,17 +387,25 @@ func (s *Server) processFile(ctx context.Context, pushEvent vcs.PushEvent, repo
 	}
 
 	// Create schema update issue
+	issueName := fmt.Sprintf("%s by %s", migrationDescription, strings.TrimPrefix(file, repo.BaseDirectory+"/"))
+	if err := s.createIssueFromMigrationFile(ctx, issueName, pushEvent, repo.ProjectID, migrationInfo.Type, migrationDetailList); err != nil {
+		return "", false, activityCreateList, echo.NewHTTPError(http.StatusInternalServerError, "Failed to create issue").SetInternal(err)
+	}
+
+	return fmt.Sprintf("Created issue %q from file %q", issueName, file), true, activityCreateList, nil
+}
+
+func (s *Server) createIssueFromMigrationFile(ctx context.Context, issueName string, pushEvent vcs.PushEvent, projectID int, migrationType db.MigrationType, migrationDetailList []*api.MigrationDetail) error {
 	creatorID := api.SystemBotID
```

Comment:
The issue creator should be derived outside of the createIssueFromMigrationFile(), and once.

Reply:
Done.